docs/3.0.0-beta.x/community/contribute-with-docs.md

```diff
@@ -0,0 +1,486 @@
+
+You are welcome to create tutorials, articles, and videos for your blog or channels without speaking to a maintainer.
+
+## General Guidelines
+
+Please review the following guidelines and recommendations regarding contributing to Docs.
+
+Broadly speaking, if you want to contribute documentation, you will either:
+
+1. Create **new** documentation for a topic not documented, or
+2. Improve existing documentation with additional examples, adding more context, fixing grammar and typos, or doing [translations](/3.0.0-beta.x/community/contribute-with-docs.html#translations).
+
+Each of the two ways above have specific [workflows](/3.0.0-beta.x/community/contribute-with-docs.html#the-workflow) which should be followed to ensure maximum efficiency in your contributions to the documentation.
+
+## Categories of Documentation
+
+There are four different **types of content** existing in the documentation:
+
+a. **Tutorials**
+
+b. **How-tos**
+
+c. **Reference Topics**
+
+d. **Explanations**
+
+Below are explanations for each category of content. The keys are to **know your audience** and **write for them**, both of which are in this contributing guide.
+
+### Tutorials
+
+A **Tutorial** is a complete set of instructions that take the reader through to the completion of a specific end, such as a project or complete working application. The purpose of tutorials is to **teach** general concepts and show how to accomplish specific actions using Strapi.
+
+This _audience member_ can be a developer who is generally speaking is new to JavaScript programming or a developer who is new to programming with Strapi. **Tutorials** are for the newer programmer. More experienced developers, who are new to Strapi, find the **How-tos** better suited to their experience, but they can also follow tutorials.
+
+Examples include the [Building a Static Blog using Gatsby and Strapi](https://blog.strapi.io/building-a-static-website-using-gatsby-and-strapi/) tutorial and the [Cooking a Deliveroo Clone](https://blog.strapi.io/strapi-next-setup/) tutorial.
+
+### How-to's
+
+A **How-to** is a **step-by-step guide showing how** to solve a specific problem or accomplish a specific task. These guides provide concise action steps and assume a general understanding of the context by the audience.
+
+This _audience member_ is a developer who is newer to programming with Strapi. General knowledge of the technology covered is assumed. Here the **How-tos** are meant to show how to do things with Strapi, without necessarily explaining the theory.
+
+Examples include the [Digital Ocean Deployment Guide](/3.0.0-beta.x/guides/deployment.html#digital-ocean) and [How to install MongoDB Locally](/3.0.0-beta.x/guides/databases.html#install-mongodb-locally).
+
+### Reference topics
+
+**Reference topics** are subjects that **are resources** to bookmark and used as a reference. Developers can be expected to refer to Reference topics regularly throughout the development of Strapi based projects.
```

Comment:
As I am reading these sections I am thinking that this should go into some Glossary section since they are definitions.

Reply:
@soupette I can see what you mean, but strictly speaking, these are categories for the documentation. I can see what you mean though that these could be considered definitions,  I think that definitions/glossary (compared to these instructions) is a little different in purpose.

As people are reading how to contribute to the documentation, it is important to layout the categories. Perhaps they could **ALSO** go in a Glossary but I wouldn't comprehend moving them completely from this section.